cloud/pkg/cloudhub/handler/messagehandler.go

```diff
@@ -456,6 +456,7 @@ func (mh *MessageHandle) MessageWriteLoop(info *model.HubInfo, stopServe chan Ex
 			conn, ok := mh.nodeConns.Load(info.NodeID)
 			if !ok {
 				if retry == 1 {
+					nodeQueue.AddRateLimited(key.(string))
```

Comment:
Thanks for the fixing! And in active-active cloudcore mode, cloudcore will fetch all edge nodes related objects, and cloudcore will omit the object if edgenode haven't connect to the related cloudcore. So here we need to break.

And in synccontroller, we will check whether oject created successfully, and if not, send the created message. But the feature still haven't implemented.

ref: https://github.com/kubeedge/kubeedge/blob/master/cloud/pkg/synccontroller/createfailedobject.go#L5

Reply:
Does this mean the proposed change breaks active-active scenarios or it just wouldn't fix the error for those but helps for single cloudcore mode? In latter case, could we merge it until a proper solution in the synccontroller is implemented?
Is there a planned timeline for the synccontroller implementation?

Reply:
This PR seems to fix at least the issue described in [#3045](https://github.com/kubeedge/kubeedge/issues/3045).

@fisherxu, I do not fully understand your comment, does this mean that [#3045](https://github.com/kubeedge/kubeedge/issues/3045) and related issues will be solved once the feature is implemented in synccontroller? Is there any chance using this PR as transistion solution until the feature in synccontroller is implemented or would it break other things?

Reply:
@fisherxu, any updates on this? Will the full solution in synccontroller be implemented in KubeEdge 1.19, oder can we use @elrosar's solution in the meantime?

Reply:
We use now cloudcore 1.9 the bug still persist with this new version.
To clarify and reproduce: 
stop edgecore wait until it has k8s status "NotReady" than create a deployment. Wait for approximately 20 Seconds. Start edgecore.
expected result: pod from deployment starts after device has k8s status "Ready".
actual result: pod hangs in state pending forever.